api/representation_test.go

```diff
@@ -546,474 +374,3 @@ func TestTaskResponse_String(t *testing.T) {
 		`"source_input":{"services":{"regexp":""}},"version":"","working_dir":"sync-tasks/task"}}`
 	require.Equal(t, expected, actual)
 }
-
-func TestTaskResponse_taskResponseFromConfigTaskConfig(t *testing.T) {
```

Comment:
Not necessary for this PR since I know this is the rebase - I'm probably missing some context. Looks like this set of tests got deleted but it looks like this function still exists (renamed to `taskResponseFromTaskConfig`)?

Reply:
Thank you for the review. Added it back + updated it! The corresponding function was removed early in the refactor effort but was later added back for `config.TaskConfig` and overlooked testing for it.